packages/myst-frontmatter/src/licenses/licenses.yml

```diff
@@ -280,6 +280,53 @@ cases:
         content:
           url: https://example.com
     warnings: 1
+  - title: url string coerces to license from spdx list
+    raw:
+      license: https://creativecommons.org/licenses/by/4.0/
+    normalized:
+      license:
+        content:
+          id: CC-BY-4.0
+          name: Creative Commons Attribution 4.0 International
+          url: https://creativecommons.org/licenses/by/4.0/
+          CC: true
+          free: true
+  - title: url coerces to license from spdx list
+    raw:
+      license:
+        url: https://creativecommons.org/licenses/by/4.0/
```

Comment:
However, this would fail - i.e. if the url was explicitly provided as `license.url`, it would _not_ validate to the SPDX license.

Now, this coerces correctly to SPDX.